### ISSUE TITLE
puzzled by the results of cntk

### ISSUE THREAD

**Issue by ee2ace** [Microsoft/CNTK#1316]
Hi,

I just want to move from my own one-thread c++ deep learning codes into the cntk, believing it will give me more efficiency.
I used the binary package to run a simple one-hidden layer network, here is the code of the model:
>>
    features = Input (116)
    labels   = Input (2)

    # model
    featScale = 100
    model = Sequential (
        Scale {featScale} :
        DenseLayer {64} : ReLU : 
        LinearLayer {2} 
    }
    
    # output
    z = model (features)

    # loss and error computation
    ce   = CrossEntropyWithSoftmax (labels, z)
    errs = ClassificationError (labels, z)

    # configuration parameters of the SGD procedure
    SGD = [
        epochSize = 0                   # =0 means size of the training set
        minibatchSize = 20
        learningRatesPerSample = 0.04   # gradient contribution from each sample
        maxEpochs = 50
    ]

and the result, I was so confused about it.
I have two group of data, one is 68 x 116 or training, another is 96 x 116 for predicting.
In my own network, the training error is 0, and the predicting error may be 20.8%.
But from cntk, their errors are all 50%.
So, where am I wrong??
I've attached the data also with the training and test results below. 

Waiting for your helps...
Thanks in advanced!

[train2.cntk.txt](https://github.com/Microsoft/CNTK/files/715515/train2.cntk.txt)
[test2.cntk.txt](https://github.com/Microsoft/CNTK/files/715516/test2.cntk.txt)
[oneHid.test.out.txt](https://github.com/Microsoft/CNTK/files/715513/oneHid.test.out.txt)
[oneHid.train.out.txt](https://github.com/Microsoft/CNTK/files/715514/oneHid.train.out.txt)

**Comment by sayanpa** [comment]
Can you please try out with a larger data set and report the error rates? The model complexity vs amount of data you have might be leading to your observation.

**Comment by raaaar** [comment]
FWIW, it's pretty unusual to have a training set that's only 70% of the test set, [common proportions are 70%/30% training/validation](https://en.wikipedia.org/wiki/Test_set#Validation_set). 

Closing this for now, but feel free to reopen, if you still observe this behavior with larger and properly split sets.

**Comment by ee2ace** [comment]
@sayanpa , I wish I could have a larger data set, but, in fMRI, to to a scan is not so easy, it's expensive, in money and time.
@raaaar , actually, the training set or the test set are not split, they are from different labs, scanning from different time, I need to know if training one group from one lab can predict another group from another lab.
